### PR TITLE
Font size for time display

### PR DESCRIPTION
**Pull request by idrak888**
Changed font size for second half of time display by using .split(".") and wrapping in separate span tags

**What does this PR do?**
Minor visual enhancement where the solve time's decimal component is a smaller font size.

**Related Issue(s)**
None

**Changes**
Used the split function to separate the time string and rendered each part in different span tags with a different font size. This is applied on 3 different places, the solve modal, the grid view and the main Timer.tsx file. 

**Additional Comments**
I haven't thoroughly tested this yet so would like some feedback and expect some adjustments.

**By submitting this PR, I confirm that:**
- [*] I have reviewed my code and believe it is ready for merging.
- [*] I understand that this PR may be subject to review and changes.
- [*] I agree to abide by the code of conduct and contributing guidelines of this project.

### CODE REVIEW

**Comment by idrak888** [comment]
@bryanlundberg please have a look

File: src/components/solves/ModalSolve.tsx [hunk 1]
```diff
@@ -75,7 +75,9 @@ export default function ModalSolve() {
         <div className="w-full h-auto text-xs border rounded-md sm:w-96 bg-neutral-200 border-neutral-800 ">
           <div className="flex items-center justify-between p-3 border-b border-neutral-400">
             <div className="flex items-center text-lg font-bold">
-              {formatTime(solve.time)}
+              <div>
+                <span>{formatTime(solve.time).split(".")[0]}</span>.<span className="sm:text-2xl md:text-3xl lg:text-5xl">{formatTime(solve.time).split(".")[1]}</span>
```
Comment (bryanlundberg) [inline src/components/solves/ModalSolve.tsx:79]: Define responsive text-size for left side of the time.

File: src/components/solves/SingleSolveItem.tsx [hunk 2]
```diff
@@ -18,7 +18,9 @@ export default function SingleSolveItem({ solve }: SingleSolveItem) {
         }}
         className="relative flex items-center justify-center w-auto p-1 text-lg font-medium text-center transition duration-200 rounded-md cursor-pointer z-1 h-14 light:bg-neutral-50 light:shadow-sm light:shadow-neutral-400 light:hover:bg-neutral-100 light:text-zinc-800 dark:bg-zinc-900 dark:hover:bg-zinc-800 dark:shadow-sm dark:text-neutral-200"
       >
-        {formatTime(solve.time)}
+        <div>
+          <span>{formatTime(solve.time).split(".")[0]}</span>.<span className="sm:text-2xl md:text-3xl lg:text-5xl">{formatTime(solve.time).split(".")[1]}</span>
```
Comment (bryanlundberg) [inline src/components/solves/SingleSolveItem.tsx:22]: Define responsive text-size for left side of the time.

File: src/components/timer/Timer.tsx [hunk 3]
```diff
@@ -51,7 +51,9 @@ export default function Timer() {
                 {translation.timer["solving"][lang]}
               </span>
             ) : (
-              formatTime(solvingTime)
+              <div>
+                <span>{formatTime(solvingTime).split(".")[0]}</span>.<span className="sm:text-2xl md:text-3xl lg:text-5xl">{formatTime(solvingTime).split(".")[1]}</span>
```
Comment (bryanlundberg) [inline src/components/timer/Timer.tsx:55]: Define responsive text-size for left side of the time.

**Review by bryanlundberg — CHANGES_REQUESTED** [review]
Hey, looks good the code implementation but the styles applied are very big.

image

image


Would be possible lower down the size of the dot?
image

**Comment by idrak888** [comment]
Okay will work on those

**Comment by idrak888** [comment]
@bryanlundberg made some adjustments, could you have a look again?

**Review by bryanlundberg — APPROVED** [review]

**Comment by bryanlundberg** [comment]
@idrak888 You did a great work, i just provided some small changes to align with the standard values i typically use. Thanks for your collaboration!

**Pull request merged**